src/tileset.rs

```diff
@@ -167,7 +167,9 @@ impl Tileset<RawPixels> {
     ) -> Result<Tileset<RawPixels>> {
         let mut reader = AseReader::new(data);
         let id = reader.dword()?;
-        let flags = reader.dword().map(|val| TilesetFlags { bits: val })?;
+        let flags = reader.dword().map(TilesetFlags::from_bits)?.ok_or(
```

Comment:
I don't think causing an error is a good idea, because it's not future-compatible.

I think we should just set the unknown bits to zero.

Reply:
As in `TilesetFlags::EMPTY_TILE_IS_ID_ZERO`?